### PR TITLE
Ambertvu/property editor

### PR DESCRIPTION
**Pull request by Ambertvu**
- Duplicated part-2.md to create part-2-v9.md
- Updated part2-md documentation as it was obsolete. It now needs a extra parameter of IEditorConfigurationParser. (only I have no idea what we can do with this yet, but this makes the example work again without giving obsolete errors)

### CODE REVIEW

File: Tutorials/Creating-a-Property-Editor/part-2.md [hunk 1]
```diff
@@ -65,18 +65,18 @@ namespace Umbraco.Cms.Core.PropertyEditors
 {
```
Comment (eshanrnh) [inline Tutorials/Creating-a-Property-Editor/part-2.md:65]: using Umbraco.Cms.Core.Services; namespace needs to be added.

File: Tutorials/Creating-a-Property-Editor/part-2.md [hunk 2]
```diff
@@ -89,15 +89,16 @@ namespace Umbraco.Cms.Core.PropertyEditors
...
+        private readonly IEditorConfigurationParser _editorConfigurationParser;
+
         public Suggestions(IDataValueEditorFactory dataValueEditorFactory,
-            IIOHelper ioHelper)
+            IIOHelper ioHelper,
+            IEditorConfigurationParser editorConfigurationParser)
             : base(dataValueEditorFactory)
-
         {
             _ioHelper = ioHelper;
```
Comment (eshanrnh) [inline Tutorials/Creating-a-Property-Editor/part-2.md:99]:
```suggestion
            _ioHelper = ioHelper;
            _editorConfigurationParser = editorConfigurationParser;
```

**Review by eshanrnh — COMMENTED** [review]
The changes look good to me and the code works fine, @Ambertvu 🙌 Just added a couple of suggestions below.

Also, can I suggest adding the code snippets in the current file under the heading version 10 (similar to [Date](https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Date/#for-version-9) article) rather than creating duplicate files just for 2 code snippets?

File: Tutorials/Creating-a-Property-Editor/part-2.md (hunk 2 above)
Comment (Ambertvu) [reply Tutorials/Creating-a-Property-Editor/part-2.md:99]: Missed this one, sharp!

**Comment by Ambertvu** [comment]
@eshanrnh Sharp eye! Processed your changes and followed your suggestions, I'll keep it in mind for future doc changes!

**Pull request merged**